src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVCleanPt1a.java

```diff
@@ -0,0 +1,439 @@
+import htsjdk.variant.vcf.VCFHeaderLineType;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.StructuralVariantDiscoveryProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFConstants;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.Map;
+import java.util.LinkedHashSet;
+import java.util.HashSet;
+import java.util.HashMap;
+import java.util.stream.Collectors;
+
+/**
+ * Completes an initial series of cleaning steps for a VCF produced by the GATK-SV pipeline.
+ *
+ * <h3>Inputs</h3>
+ * <ul>
+ *     <li>
+ *         VCF containing structural variant (SV) records from the GATK-SV pipeline.
+ *     </li>
+ *     <li>
+ *         TODO
+ *     </li>
+ * </ul>
+ *
+ * <h3>Output</h3>
+ * <ul>
+ *     <li>
+ *         Annotated VCF.
+ *     </li>
+ * </ul>
+ *
+ * <h3>Usage Example</h3>
+ * <pre>
+ *     gatk SVCleanPt1a \
+ *       -V structural.vcf.gz \
+ *       -O cleansed.vcf.gz
+ *       --ped-file pedigree.ped
+ *       --chrX chrX
+ *       --chrY chrY
+ *       --fail-list background_fail.txt
+ *       --pass-list bothsides_pass.txt
+ *       --sample-list sample_list.txt
+ *       --revised-list revised_list.txt
+ * </pre>
+ *
+ * <h3>Cleaning Steps</h3>
+ * <ol>
+ *     <li>
+ *         Adds new FILTER and INFO tags to header.
+ *     </li>
+ *     <li>
+ *         TODO
+ *     </li>
+ * </ol>
+ */
+@CommandLineProgramProperties(
+        summary = "Clean and format structural variant VCFs per Step 1a",
+        oneLineSummary = "Clean and format structural variant VCFs per Step 1a",
+        programGroup = StructuralVariantDiscoveryProgramGroup.class
+)
+@BetaFeature
+@DocumentedFeature
+public final class SVCleanPt1a extends VariantWalker {
+    public static final String PED_FILE_LONG_NAME = "ped-file";
+    public static final String CHRX_LONG_NAME = "chrX";
+    public static final String CHRY_LONG_NAME = "chrY";
+    public static final String FAIL_LIST_LONG_NAME = "fail-list";
+    public static final String PASS_LIST_LONG_NAME = "pass-list";
+    public static final String OUTPUT_SAMPLES_LIST_LONG_NAME = "sample-list";
+    public static final String OUTPUT_REVISED_EVENTS_LIST_LONG_NAME = "revised-list";
+
+    @Argument(
+            fullName = PED_FILE_LONG_NAME,
+            doc = "Sample PED file"
+    )
+    private GATKPath pedFile;
```

Comment:
No need for this if you've already run the gatk formatting script on the vcf. Each genotype will have an `ECN` FORMAT field with the ploidy that you can check to infer sex.

Reply:
Updated accordingly - thanks for the pointer here. Seems suitable since we only use sex information when processing variants along chrX / chrY.